code/modules/projectiles/projectile/change.dm

```diff
@@ -16,7 +16,7 @@
 
 /obj/item/projectile/change/proc/wabbajack(var/mob/living/M,var/type) //WHY: as mob in living_mob_list
 	if(istype(M, /mob/living) && M.stat != DEAD)
-		if(istype(M, /mob/living/carbon/human/manifested)) // DEEEEEEEEEEEEEEITY
+		if(ismanifested(M))
```

Comment:
DEEEEEEEEEEEEEEITY

Reply:
who even wrote that and why?

Reply:
Granted I wonder if it's still a needed check considering that manifested ghosts aren't simple to produce en masse anymore.